packages/inter-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -1071,6 +1037,14 @@ export const prepareVaultManagerKit = (
             /** @type {import('@agoric/zoe/src/contractSupport/atomicTransfer.js').TransferPart[]} */
             const transfers = [];
             let liquidated = 0;
+            /** @type {MapStore<string, Vault>} */
+            const reinstateVaults = makeScalarMapStore();
```

Comment:
verb phrase for a store is awkward.
```suggestion
            const vaultsToReinstante = makeScalarMapStore();
```

Reply:
done